api/openapi.yaml

```diff
@@ -1023,8 +1023,8 @@ paths:
         default:
           $ref: "#/components/responses/UnexpectedProblemResponse"
   # TODO: should this be /value to be compatible with all three entitlement types
-  /api/v1/subjects/{subjectIdOrKey}/entitlements/{entitlementIdOrFeatureKey}/value:
-    get:
+  ? /api/v1/subjects/{subjectIdOrKey}/entitlements/{entitlementIdOrFeatureKey}/value
+  : get:
```

Comment:
?

Reply:
My editor messes with me, I will sort this out asap.

Reply:
Seems like VSCode tries to wrap lines with this syntax if they are more than 80 characters, updated my settings.